src/Avalonia.Base/Threading/AvaloniaSynchronizationContext.cs

```diff
@@ -28,13 +28,13 @@ public static void InstallIfNeeded()
         }
 
         /// <inheritdoc/>
-        public override void Post(SendOrPostCallback d, object state)
+        public override void Post(SendOrPostCallback d, object? state)
         {
            Dispatcher.UIThread.Post(() => d(state), DispatcherPriority.Send);
         }
```

Comment:
Unrelated, but this overload with "object state" looks useless, it still creates closure for "state" obj.